en/releases/release-1.2.0-beta.2.md

```diff
@@ -0,0 +1,34 @@
+---
+title: TiDB Operator 1.2.0-beta.2 Release Notes
+---
+
+# TiDB Operator 1.2.0-beta.2 Release Notes
+
+Release date: April 29, 2021
+
+TiDB Operator version: 1.2.0-beta.2
+
+## Rolling Update Changes
+
+- Upgrading TiDB Operator will cause the recreation of the TidbMonitor Pod due to [#3943](https://github.com/pingcap/tidb-operator/pull/3943)
+- Upgrading TiDB Operator will cause the recreation of the DM-master Pod due to [#3914](https://github.com/pingcap/tidb-operator/pull/3914)
+
+## New Features
+
+- TidbMonitor supports monitoring multiple TidbCluster with TLS enabled ([#3867](https://github.com/pingcap/tidb-operator/pull/3867), [@mikechengwei](https://github.com/mikechengwei))
+- Support configuring `podSecurityContext` for all TiDB components ([#3909](https://github.com/pingcap/tidb-operator/pull/3909), [@liubog2008](https://github.com/liubog2008))
+- Support configuring `topologySpreadConstraints` for all TiDB components ([#3937](https://github.com/pingcap/tidb-operator/pull/3937), [@liubog2008](https://github.com/liubog2008))
+- Support deploying DmCluster in a different namespace of a TidbCluster ([#3914](https://github.com/pingcap/tidb-operator/pull/3914), [@csuzhangxc](https://github.com/csuzhangxc))
+- Support installing TiDB Operator with namespace scoped permissions ([#3896](https://github.com/pingcap/tidb-operator/pull/3896), [@csuzhangxc](https://github.com/csuzhangxc))
+
+## Improvements
+
+- Add readiness probe for the TidbMonitor Pod ([#3943](https://github.com/pingcap/tidb-operator/pull/3943), [@mikechengwei](https://github.com/mikechengwei))
+- Optimize TidbMonitor for DmCluster with TLS enabled ([#3942](https://github.com/pingcap/tidb-operator/pull/3942), [@mikechengwei](https://github.com/mikechengwei))
+- TidbMonitor supports not generating Prometheus alert rules ([#3932](https://github.com/pingcap/tidb-operator/pull/3932), [@mikechengwei](https://github.com/mikechengwei))
+- Optimize the `PodsAreChanged` function ([#3901](https://github.com/pingcap/tidb-operator/pull/3901), [@shonge](https://github.com/shonge))
```

Comment:
This can be removed.

Reply:
removed in https://github.com/pingcap/docs-tidb-operator/pull/1196/commits/89822699bdc72b4d1f06fdbfab3aad043c65dc8b